.github/workflows/format.yml

```diff
@@ -0,0 +1,45 @@
+name: Format
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  edit:
+    name: clang-format
+    if: ${{ github.event.comment.body == '@votca-bot format' }}
+    runs-on: ubuntu-latest
+    container: votca/buildenv:latest
+    steps:
+      - name: Check if message comes from PR
+        uses: octokit/request-action@v2.x
+        id: issue_info
+        with:
+          route: GET /repos/:repository/issues/:pull_number
+          repository: ${{ github.repository }}
+          pull_number: ${{ github.event.issue.number }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Get PR info
+        if: ${{ fromJSON(steps.issue_info.outputs.data).pull_request }}
+        uses: octokit/request-action@v2.x
+        id: pr_info
+        with:
+          route: GET /repos/:repository/pulls/:pull_number
+          repository: ${{ github.repository }}
+          pull_number: ${{ github.event.issue.number }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Checkout head branch from PR
+        if: ${{ fromJSON(steps.issue_info.outputs.data).pull_request }}
+        uses: actions/checkout@master
+        with:
+          repository: ${{ fromJSON(steps.pr_info.outputs.data).head.repo.full_name }}
+          ref: ${{ fromJSON(steps.pr_info.outputs.data).head.ref }}
+      - name: Run clang-format
+        if: ${{ fromJSON(steps.issue_info.outputs.data).pull_request }}
+        run: find . -type f -regex ".*\.\(cc\|h\)" -print0 | xargs -0 clang-format -i -style=file
+      - name: Commit and push
+        if: ${{ fromJSON(steps.issue_info.outputs.data).pull_request }}
+        run: |
+          git config --global user.name "Votca Bot"
+          git config --global user.email "github@votca.org"
+          git add -u
+          git commit -m "Format code using clang-format" || true
```

Comment:
Commit message ok?

Reply:
Can we do "votca-bot: Format code using clang-format"

Reply:
@votca-bot is already the author of the commit.  Including the version seems reasonable.

Reply:
It will say "Format code using clang-format version 10.0.0 (Fedora 10.0.0-2.fc32)" now.